roles/st2mistral/tasks/main.yml

```diff
@@ -67,11 +74,19 @@
 
 - name: Register mistral actions
   become: yes
-  shell: /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate && touch /etc/mistral/mistral-db-manage.upgrade.head.ansible.has.run
+  shell: /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate && touch /etc/mistral/mistral-db-manage.populate.ansible.has.run
```

Comment:
We touch the file first, find: `&& touch /etc/mistral/mistral-db-manage.populate.ansible.has.run`
and later in block don't execute it if `touch` exists: 
```yaml
creates: /etc/mistral/mistral-db-manage.populate.ansible.has.run`
```